dts/boards: nordic: Fix reg and ranges property issues

DematteisGiacomo opened the pull request:
dts/boards: nordic: fix register properties
Add missing reg property to peripherals.



boards: nordic: nrf9280pdk: add ranges to reserved  memory
 Add missing "ranges".

boards: nordic: nrf9280pdk: remove /delete-node line   
 Remove `/delete-node/ ipc-1-3;` as this has no effect on the context.

DematteisGiacomo committed:
dts/boards: nordic: fix register properties

Add missing reg property to peripherals.

Signed-off-by: Giacomo Dematteis <giacomo.dematteis@nordicsemi.no>

DematteisGiacomo committed:
boards: nordic: nrf9280pdk: add ranges to reserved  memory

Add missing "ranges".

Signed-off-by: Giacomo Dematteis <giacomo.dematteis@nordicsemi.no>

kylebonnici commented on boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280-memory_map.dtsi:
```diff
@@ -7,6 +7,9 @@
 
 / {
 	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
```
No need to re defined these are already defined in nrf9280.dtsi

kylebonnici reviewed (approved):
My approval implies that the changes requested by the vscode team were done.  But we do not have scope and knowledge to determine  any side effects.  Please make sure someone who knows how these files are used in the SDK by apps and customers; also reviews these changes

maxd-nordic reviewed (dismissed):
lgtm

DematteisGiacomo committed:
dts: vendor: nordic: Fix missing mapping nrf9280.dtsi

Fix missing mapping for usbhs@86000 <0x2f700000 0x40000>
on parent node peripheral@5f000000.

Signed-off-by: Giacomo Dematteis <giacomo.dematteis@nordicsemi.no>

kylebonnici commented:
@DematteisGiacomo @bjarki-andreasen Any blockers on this PR? I would rely like to close off tasks on my end that depend on this being merged

kylebonnici commented on boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280-memory_map.dtsi:
```diff
@@ -7,6 +7,7 @@
 
 / {
 	reserved-memory {
+		ranges;
```
Yes I agree we should not be using `reserved-memory`, When this PR was done, we had not yet changed how partitions are done!
